.github/workflows/run-network-tests.yml

```
@@ -14,7 +14,7 @@ jobs:
       matrix:
         node-version: [14.x]
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v2
     - name: Use Node.js ${{ matrix.node-version }}
       uses: actions/setup-node@v1
```

This one was also updated to `v3` in workflows like [.github/workflows/joystream-cli.yml](https://github.com/Joystream/joystream/pull/4379/files#diff-81c9870c963d108d2e27f1ab912ca133e2b62feca653ff14f0de1266f098c5d4):
```suggestion
      uses: actions/setup-node@v3
```